cms/jinja2/templates/components/contact_details/contact_details.html

```diff
@@ -0,0 +1,21 @@
+{% from "components/text-indent/_macro.njk" import onsTextIndent %}
+
+{% if page.contact_details %}
```

Comment:
suggestion: make this more agnostic. That is, have it expect a `contact_details` variable (which is a `ContactDetails` instance

Reply:
Done.